bundles/http/Citrix/7da37e64-5472-41fc-b86d-8923dafe00e5/i18n/zh-CN.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dfd179ed-bddd-4c9b-a269-2ebeb6a797e0": {
+    "3f0a059c-4aea-453f-9e09-e4120006fb10": "URL",
+    "5fff755e-31ea-4827-933a-e34d4541eb01": "Search Dashbard",
```

Comment:
Why this string is not localized?

Reply:
this is a test PR. actual translations are in progress.